db/controllers/controller.js

```diff
@@ -12,6 +15,47 @@ const getTopics = (req, res, next) => {
   });
 };
 
+const postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+  checkArticleExists(article_id)
+    .then((exists) => {
```

Comment:
This controller is a little more complicated than necessary. I would expect several of these errors to be rejected by pSQL when we tried to make the request. Make sure you run your tests first and check the output in case the promise has already been rejected by pSQL